src/components/slider/slider.component.ts

```diff
@@ -220,36 +228,39 @@ export class SliderComponent implements OnInit, AfterViewInit, DoCheck {
 
     thumbEvent(thumb: SliderThumb, event: SliderThumbEvent): void {
 
-        // get the current thumb state
-        const state = this.getThumbState(thumb);
+        if (!this.disabled) {
```

Comment:
Usually with disabled-type properties, we do:

```ts
if (this.disabled) {
  return;
}
```

at the top of the function. This helps reduce nesting, which can be problematic especially in large functions like this.